ts/test/BUILD.bazel

```diff
@@ -0,0 +1,74 @@
+load("@bazel_skylib//lib:partial.bzl", "partial")
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("@aspect_rules_swc//swc:defs.bzl", "swc_transpiler")
+load(":transpiler_tests.bzl", "transpiler_test_suite")
+load("//ts:defs.bzl", "ts_project")
+
+_TSCONFIG = {
+    "compilerOptions": {
+        "declaration": True,
+        "sourceMap": True,
+    },
+}
+
+_transpiler = partial.make(
+    swc_transpiler,
+    source_maps = "true",
+)
+
+write_file(
+    name = "gen_ts",
+    out = "big.ts",
+    content = [
+        "export const a{0}: number = {0}".format(x)
+        for x in range(1000)
+    ],
+)
+
+write_file(
+    name = "gen_typeerror",
+    out = "typeerror.ts",
+    content = ["export const a: string = 1"],
+)
+
+write_file(
+    name = "gen_lib_dts",
+    out = "lib.d.ts",
+    content = ["export const a: string;"],
+)
+
+write_file(
+    name = "gen_index_ts",
+    out = "index.ts",
+    content = ["export const a: string = \"1\";"],
+)
+
+ts_project(
+    name = "transpile_with_swc",
+    srcs = ["big.ts"],
+    transpiler = _transpiler,
+    tsconfig = _TSCONFIG,
+)
+
+ts_project(
+    name = "transpile_with_typeerror",
+    srcs = ["typeerror.ts"],
+    # The transpile_with_typeerror.check target will have a build failure
+    # But the default transpile_with_typeerror target should still produce JS outs
```

Comment:
anything downstream that consumes typeerror.js to make sure it is emitted? a js_binary perhaps?

Reply:
oh I see, it is checked in transpiler_test_suite(). maybe plumb it through to a js_test as well to be extra clear and add additional coverage?

Reply:
it's in the transpiler_tests.bzl, you'll get there in a moment :)

Reply:
sure can add build_test